apps/demo/app/custom-ui/[...puckPath]/client.tsx

```diff
@@ -314,6 +314,10 @@ export function Client({ path, isEdit }: { path: string; isEdit: boolean }) {
         ...config.components.Hero,
         permissions: {
           debug: false,
+          drag: false,
+          delete: false,
+          duplicate: false,
+          insert: false,
```

Comment:
This seems to be disabling the `insert` behaviour of all items in the custom UI, not just the Hero

Reply:
@xaviemirmon something weird is going on here... steps to reproduce in custom UI demo

1. Drag in a non-disabled item
2. Disabled item (Hero) becomes enabled
3. Drag in Hero
4. All items now become disabled